melange-fest.opam

```diff
@@ -21,6 +21,7 @@ depends: [
   "melange-testing-library" {with-test}
   "ocaml-lsp-server" {with-test}
   "odoc" {with-doc}
+  "ocamlformat"
```

Comment:
Can we add here a `with-doc` or `with-test` constraint? We don't want consumers of the library to install this package as it's not really a requirement to use the library.

We can upgrade to `with-dev-setup` once opam 2.2 becomes available.